runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/KafkaStreamingTest.java

```diff
@@ -53,32 +56,14 @@
  */
 public class KafkaStreamingTest {
   private static final EmbeddedKafkaCluster.EmbeddedZookeeper EMBEDDED_ZOOKEEPER =
-          new EmbeddedKafkaCluster.EmbeddedZookeeper();
+      new EmbeddedKafkaCluster.EmbeddedZookeeper();
```

Comment:
Now that you support `Read.Unbounded` can you use `KafkaIO` ROS tests? I know they have an unclear interpretation (they only work with embedded Kafka & embedded runner, last time I was involved) but seems to apply to this case.